src/cloudflare/BUILD.bazel

```diff
@@ -0,0 +1,65 @@
+load("@aspect_rules_ts//ts:defs.bzl", "ts_config", "ts_project")
+load("@workerd//:build/wd_api_bundle.bzl", "wd_api_bundle")
+load("@npm//:eslint/package_json.bzl", eslint_bin = "bin")
+
+modules = glob(["*.ts"])
+js_modules = glob(["*.js"])
+
+internal_modules = glob(["internal/*.ts"])
+internal_js_modules = glob(["internal/*.js"])
+
+srcs = modules + internal_modules
+
+eslintrc_json = ":.eslintrc.json"
+
+tsconfig_json = ":tsconfig.json"
+
+ts_config(
```

Comment:
I'll add a todo to extract a rule for these 4 things together like we do upstream.